src/multiselect_shortcut.js

```diff
@@ -94,9 +94,9 @@ const registerShortcutDelete = function() {
   };
   Blockly.ShortcutRegistry.registry.register(deleteShortcut);
   Blockly.ShortcutRegistry.registry.addKeyMapping(
-      Blockly.utils.KeyCodes.DELETE, deleteShortcut.name);
+      Blockly.utils.KeyCodes.DELETE, "multiselectDelete", true);
```

Comment:
Why are you replacing `deleteShortcut.name` with the literal string here? It should still work without the change (as you have already changed the `name` property above).